lib/adhearsion-ivr/ivr_controller.rb

```diff
@@ -231,5 +237,10 @@ def completion_callback
     def failure_callback
       instance_exec &self.class.failure_callback if self.class.failure_callback
     end
+
+    # Called to wrap a prompt in a hash signaling it is actually an url
```

Comment:
I don't really care, as a user of adhearsion-ivr, that this wraps a prompt in a hash. What I care about is that this causes the prompt to be fetched from the specified URL by the engine rather than sent in full. Could you correct the docs to match?